snuba/cli/replacer.py

```diff
@@ -91,19 +99,26 @@ def replacer(
     setup_logging(log_level)
     setup_sentry()
 
-    dataset = get_dataset(dataset_name)
+    storage = get_writable_storage(storage_name)
```

Comment:
you can do this after the if block below. it would be more evident

Reply:
It's not the same since storage gets a default value. If dataset is provided it must take priority.